pom.xml

```diff
@@ -607,6 +637,10 @@
                     <!--runOrder>alphabetical</runOrder-->
                     <useFile>false</useFile>
                     <enableAssertions>true</enableAssertions>
+                    <!-- https://stackoverflow.com/questions/53437819/maven-surefire-and-jdk-11-->
+                    <!-- https://winterbe.com/posts/2018/08/29/migrate-maven-projects-to-java-11-jigsaw/-->
+                    <forkCount>0</forkCount> <!-- changed this to 0 -->
```

Comment:
I like the comments above about the reasoning, but "changed this to 0" is merely stating the obvious ;)

Reply:
Haha...good point there!